README.md

```diff
@@ -36,215 +51,26 @@ Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs
-```shell
-minishift service [-n NAMESPACE] [--url] NAME
-```
-
-## Networking
-
-The minishift VM is exposed to the host system via a host-only IP address, that can be obtained with the `minishift ip` command.
-Any services of type `NodePort` can be accessed over that IP address, on the NodePort.
-
-To determine the NodePort for your service, you can use a `kubectl` command like this:
-
-`kubectl get service $SERVICE --output='jsonpath="{.spec.ports[0].NodePort}"'`
-
-## Persistent Volumes
-
-Minishift supports [PersistentVolumes](http://kubernetes.io/docs/user-guide/persistent-volumes/) of type `hostPath`.
-These PersistentVolumes are mapped to a directory inside the minishift VM.
-
-The MiniShift VM boots into a tmpfs, so most directories will not be persisted across reboots (`minishift stop`).
-However, MiniShift is configured to persist files stored under the following host directories:
-
-* `/data`
-* `/var/lib/minishift`
-* `/var/lib/docker`
-
-Here is an example PersistentVolume config to persist data in the '/data' directory:
-
-```yaml
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: pv0001
-spec:
-  accessModes:
-    - ReadWriteOnce
-  capacity:
-    storage: 5Gi
-  hostPath:
-    path: /data/pv0001/
-```
-
-## Mounted Host Folders
-Some of drivers will mount a host folder within the VM so that you can easily share files between the VM and host.  These are no configurable at the moment and different for the driver and OS you are using.  Note: Host folder sharing is not implemented in the KVM driver yet.
-
-| Driver | OS | HostFolder | VM |
-| --- | --- | --- | --- |
-| Virtualbox | Linux | /home | /hosthome |
-| Virtualbox | OSX | /Users | /Users |
-| Virtualbox | Windows | C://Users | /c/Users |
-| VMWare Fusion | OSX | /Users | /Users |
-| Xhyve | OSX | /Users | /Users |
-
-
-## Private Container Registries
-
-To access a private container registry, follow the steps on [this page](http://kubernetes.io/docs/user-guide/images/).
-
-We recommend you use ImagePullSecrets, but if you would like to configure access on the minishift VM you can place the `.dockercfg` in the `/home/docker` directory or the `config.json` in the `/home/docker/.docker` directory.
+You can follow the [Quickstart tutorial](/docs/get-started/quickstart.md) to set up
+and start Minishift with basic configuration.
 
 ## Documentation
-For a list of minishift's available commands see the [full CLI docs](https://github.com/jimmidyson/minishift/blob/master/docs/minishift.md).
 
-## Known Issues
-* Features that require a Cloud Provider will not work in Minishift. These include:
-    * LoadBalancers
-    * PersistentVolumes
-    * Ingress
-* Features that require multiple nodes. These include:
-    * Advanced scheduling policies
-* Alternate runtimes, like rkt.
+The following documentation is available:
 
-## Design
+* [Getting Started](/docs/get-started)
```

Comment:
How are these links supposed to work? They are just pointing to a directory with no file in it (like README) which gets rendered by default. So now you have just a directory listing of markdown files. Same for "Using Minishift" and "Developing Minishift"